src/main/java/loanbook/logic/SendReminder.java

```diff
@@ -0,0 +1,84 @@
+package loanbook.logic;
+
+import java.util.Date;
+import java.util.Properties;
+
+import javax.mail.Session;
+import javax.mail.Transport;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeMessage;
+
+import loanbook.model.Model;
+import loanbook.model.loan.Loan;
+
+/**
+ * The class is used for create reminder email and send the email.
+ */
+public class SendReminder {
+    private String myEmailAccount;
+    private String myEmailPassword;
+    private String myEmailSmtpHost = "smtp.gmail.com";
+    private Loan targetLoan;
+
+    public SendReminder(Model model, String myEmailPassword, Loan targetLoan) {
+        this.myEmailAccount = model.getMyEmail();
+        this.myEmailPassword = myEmailPassword;
+        this.targetLoan = targetLoan;
+    }
+
+    /**
+     * Send a reminder email
+     *
+     * @throws Exception
+     */
+    public void send() throws Exception {
+        Properties props = new Properties();
+        props.setProperty("mail.transport.protocol", "smtp");
+        props.setProperty("mail.smtp.host", myEmailSmtpHost);
+        props.setProperty("mail.smtp.auth", "true");
+
+        final String smtpPort = "465";
+        props.setProperty("mail.smtp.port", smtpPort);
+        props.setProperty("mail.smtp.socketFactory.class", "javax.net.ssl.SSLSocketFactory");
+        props.setProperty("mail.smtp.socketFactory.fallback", "false");
+        props.setProperty("mail.smtp.socketFactory.port", smtpPort);
+
+        Session session = Session.getInstance(props);
+        session.setDebug(true);
+
+        MimeMessage message = createReminderEmail(session, myEmailAccount);
+
+        Transport transport = session.getTransport();
+        transport.connect(myEmailAccount, myEmailPassword);
+        transport.sendMessage(message, message.getAllRecipients());
+        transport.close();
+    }
+
+    /**
+     * Create a reminder email
+     *
+     * @throws Exception
+     */
+    public MimeMessage createReminderEmail(Session session, String sendMail) throws Exception {
```

Comment:
Is it possible to throw a more specific Exception?

Reply:
Fixed. Will push later after you finish reviewing this. Thank you ;)